js/main.js

```diff
@@ -114,10 +114,9 @@ function populateBudget() {
 
 function handleTripSelection(event) {
   event.preventDefault();
-  if ($tripSelect.value === "Select") {
-    switchView('home');
-  } else {
-    switchView($tripSelect.value)
+  switchView('home');
+  if ($tripSelect.value !== 'Select') {
+    switchView($tripSelect.value);
   }
```

Comment:
This isn't what I meant. You should make the value of the `value` attribute on this option `"home"` so that whenever the function is called, you can always just use the input value to get the name of the view to go to.